NativeApp/src/components/Common/CustomDatePickerIOS/index.js

```diff
@@ -19,7 +19,7 @@ class CustomDatePickerIOS extends Component {
   };
 
   static defaultProps = {
-    minimumDate: '',
+    minimumDate: moment().format('YYYY-MM-DD'),
```

Comment:
am I right in saying that moment will return the current date here? - if so we will not be able to retroactively book days off, is this expected behaviour?

Reply:
in my previous experience with Liberty's system you could book days retroactively, e.g. if you last minute booked off a day and needed to book it into the system after you come back - but that's a decision for the BA i reckon

Reply:
Didn't think of that. I was just basing on having to book holidays in advance. It's an easy change if it needs updated, so gonna merge this. :)